src/routes/preHandlers/users.js

```diff
@@ -108,6 +109,9 @@ exports.authorizeUserUpdate = async (req) => {
     checkUpdateAndCreateRestrictions(req.payload);
   }
 
+  if (get(req.payload, 'contact.countryCode') && !get(req.payload, 'contact.phone')) throw Boom.badData();
```

Comment:
Ces conditions sont redondantes avec le and sur la route non ?

Reply:
c'est justement pour pallier le cas où l'on envoie les 2 mais que l'un contient une string vide